Tribler/Test/test_as_server.py

```diff
@@ -334,6 +334,10 @@ def setUp(self):
         self.assertFalse(Session.has_instance(), 'A session instance is already present when setting up the test')
         AbstractServer.setUp(self, annotate=False)
 
+        if sys.platform == 'linux2' and os.environ.get("TRIBLER_INITTHREADS", "true").lower() == "true":
```

Comment:
Looking good!
Move the if statements to the initialize method too and I'll merge it.